docs/security/outbound-requests/telemetry.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Telemetry
+description: Telemetry from Octopus Server and what is included
+position: 25
+---
+
+Telemetry reporting is enabled by default. We use the data we receive to inform our decisions and to help us make a better product. You can opt-out of sending telemetry by navigating to **{{Configuration,Telemetry}}** in Octopus.
+
+When the "Telemetry Reporting" option is enabled, Octopus will make a HTTPS request, this request includes:
+
+- The current Octopus Deploy version number that you are running.
+- A unique installation ID (read more below).
+- Telemetry data payload
+
+When the "Telemetry Reporting" option is enabled, we'll send some specific aggregate criteria along with the request. This has evolved a bit over time, so it depends on the version you are running:
```

Comment:
The wording `When the "Telemetry Reporting" option is enabled` feels repetitive given its mentioned verbatim just before the preceding bullet points.

```suggestion
In addition, we'll send some specific aggregate criteria along with the request. This has evolved a bit over time, so it depends on the version you are running:
```